iroh-net/src/magicsock.rs

```diff
@@ -38,11 +34,9 @@ use iroh_metrics::{inc, inc_by};
 use quinn::AsyncUdpSocket;
 use rand::{seq::SliceRandom, Rng, SeedableRng};
```

Comment:
note that this is inconsistent with the rest

Reply:
formatter now takes care of this :tada: